tracks/ruby/exercises/word-count/README.md

```diff
@@ -25,14 +25,25 @@ end
 
 ```
 
+An exceptional nice solution uses `group_by(&:itself)`, and the `transform_values` method introduced in Ruby 2.4
```

Comment:
"An exceptionally nice" otherwise it is "exceptional and nice".  I don't believe it is exceptional, so the first correction is the one I think it should be.

Reply:
Thanks!